examples/src/bin/lcd_cam_ov2640.rs

```diff
@@ -0,0 +1,422 @@
+//! Drives the camera on a Freenove ESP32-S3 WROOM
+//!
+//! This example reads a JPEG from an OV2640 and writes it to the console as hex.
+//!
+//! Pins used:
+//! SIOD      GPIO4
+//! SIOC      GPIO5
+//! XCLK      GPIO15
+//! VSYNC     GPIO6
+//! HREF      GPIO7
+//! PCLK      GPIO13
+//! D0        GPIO11
```

Comment:
The main difference is the numbering scheme of the data pins, could you add something like this to address the esp32s3 eye schematic references

```
//! D0 (Y2)       GPIO11
//! D1 (Y3)       GPIO9
//! D2 (Y4)       GPIO8
//! D3 (Y5)       GPIO10
//! D4 (Y6)       GPIO12
//! D5 (Y7)       GPIO18
//! D6 (Y8)       GPIO17
//! D7 (Y9)       GPIO16
```

Reply:
The numbering scheme for my board is the same so I'll just use the 2, 3, 4 ... scheme